subtests/docker_cli/systemd/systemd.py

```diff
@@ -0,0 +1,80 @@
+"""
+Summary
+-------
+
+Operational Summary
+---------------------
+    1. Copy Unit file
+    2. Build image using predefined Dockerfile & p4321-server.py
+    3. Run image
+    4. Confirm p4321-server script is running
+    5. Stop container
+    6. cleanup
+
+"""
+
+import shutil
+import time
+from os.path import join
+from autotest.client import utils
+from autotest.client.shared.utils import is_port_free
+from dockertest.subtest import Subtest
+from dockertest.dockercmd import DockerCmd
+
+
+class systemd(Subtest):
+
+    def initialize(self):
+        super(systemd, self).initialize()
+        self.stuff['tcp_ip'] = self.config['tcp_ip']
+        self.stuff['tcp_port'] = self.config['tcp_port']
+        self.stuff['sysd_unit_file'] = self.config['sysd_unit_file']
+        self.stuff['sysd_unitf_dest'] = self.config['sysd_unitf_dest']
+        self.stuff['name'] = self.config['name']
+        self.stuff['self.tmpdir'] = self.tmpdir
```

Comment:
Yes, exactly.

IIRC I suggested using `sub_stuff` because the previous version of your PR did some combinations of `bindir` and `self.config[something]`. But in this version of your PR there's nothing like that.